src/main/views/et-fortnightly-list.njk

```diff
@@ -110,7 +110,6 @@
                       <td class="govuk-table__cell">{{ record.respondent }}<br />{{ rep }}: {{ noRep }}</td>
                     {% endif %}
                     <td class="govuk-table__cell">{{ record.hearingType }}</td>
-                    <td class="govuk-table__cell">{{ record.jurisdiction }}</td>
```

Comment:
Nothing wrong with this line here but there is a field called 'jurisdiction' for ET fortnightly list needs removing from the class EtListsService

Reply:
Ah missed this thanks kian